crs_support/urdf/swri_demo.urdf.xacro

```diff
@@ -72,7 +72,7 @@
         <geometry>
           <box size="$(arg table_size) 0.1 $(arg table_size)"/>
         </geometry>
-      </collision>
+      </collision>-->
```

Comment:
Should the walls be removed?

Reply:
I removed that wall so I could test with that vertical aircraft part we have. The virtual walls were in that exact same spot